engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbTimerInstanceState.java

```diff
@@ -31,7 +31,7 @@ public final class DbTimerInstanceState implements MutableTimerInstanceState {
   private final ColumnFamily<
           DbCompositeKey<DbLong, DbCompositeKey<DbForeignKey<DbLong>, DbLong>>, DbNil>
       dueDateColumnFamily;
-  private final DbLong dueDateKey;
```

Comment:
:thought_balloon: I guess it contains the `Key` prefix since it is part of a key no?

Reply:
Not really. First it was a suffix, not a prefix. And in other places, when the key is read out, it is also referred to as `dueDate`

```
dueDateColumnFamily.whileTrue(
        (key, nil) -> {
                 final DbLong dueDate = key.first();
```

Reply:
True, wrong wording sorry. Still it is part of the key, but ok nevermind.